benches/noria/README.md

```diff
@@ -0,0 +1,60 @@
+This directory is for running one of the main benchmarks for
+[noria](https://github.com/mit-pdos/noria), a highly concurrent
+research-prototype database that uses tokio. This is intended as a macro
+benchmark that covers many aspects of tokio's performance profile.
+
+## To run
+
+```console
+$ ./run.sh
+# go get tea/coffee
```

Comment:
lol, I briefly misunderstood this as suggesting that users run the `go get` command to get the Go package "tea/coffee". maybe i'm just a fool, but, WDYT of 
```suggestion
# this takes a while, so go get tea/coffee
```

Reply:
That is _very_ funny